### DISCUSSION TITLE
2.8.8

### DISCUSSION THREAD

**Discussion by Hunlongyu** [Hunlongyu/ZY-Player#779] [交流]
## 更新日志
1. 更新远程视频源，首次安装用户默认直接获取该远程视频源
2. 移除推荐模块
3. 移除作者赞助二维码
4. 增加软件免费说明
5. 修复部分小 bug
6. 更新依赖，让项目跑起来

### 申明
1. 软件永久开源免费！！！
2. 如果没有新的开发者加入维护更新，这将是最后一个版本，且用且珍惜。

<hr /><em>This discussion was created from the release <a href='https://github.com/Hunlongyu/ZY-Player/releases/tag/v2.8.8'>2.8.8</a>.</em>

**Comment by 419928925** [comment]
niu pi  ke la si  ,又更新了

**Comment by TeddiAngel** [comment]
2.8.8 MAC版 安装后没有视频资源,重置一次就可以了。谢谢！

**Comment by lifeisdox** [comment]
这个2.8.8版本的mac版安装后，为什么要求获取辅助功能（控制电脑）的权限？请问哪些功能需要这个权限？

**Reply by Hunlongyu** [reply]
更新了底层框架的原因吧。主要是我没有 mac 电脑，不清楚 mac 上权限问题。不过应该不大吧。

**Comment by woisdong** [comment]
为什么我的进去之后，没有看豆瓣推荐呀

**Reply by Hunlongyu** [reply]
删除了，没有人维护的话，推荐列表没有意义

**Comment by woisdong** [comment]
您好！已收到您的邮件。

**Comment by xj19930318** [comment]
没更新安卓版吗？

**Comment by woisdong** [comment]
您好！已收到您的邮件。

**Comment by Tree33-web** [comment]
MAC 2.8.8下载后无法使用，提示获取云端源站失败. Error: Request failed with status code 404
重置软件也不行

**Comment by qishuai** [comment]
你好收到您的邮件。我会尽快给您回复。谢谢，保持联系！

**Comment by woisdong** [comment]
您好！已收到您的邮件。

**Comment by Tree33-web** [comment]
软件一直显示检测到视频源未能正常加载, 即将重置源.
重置软件也不行
有没有大佬提供新的MAC视频源？

**Comment by CandiceYu0404** [comment]
<img width="1080" alt="image" src="https://github.com/Hunlongyu/ZY-Player/assets/165130590/1913d319-c2b2-41cd-b9b6-a72574dcc3b7">
请问大佬，我重制了之后也还是这个样子，应该怎么办呢？

**Reply by miyazawa-kenya** [reply]
一樣的問題，請問解決了嗎

**Comment by woisdong** [comment]
您好！已收到您的邮件。

**Comment by iced-lemon-tea** [comment]
怎么重新获取默认的远程视频源，不小心重置了一下没有源了

**Comment by qishuai** [comment]
你好收到您的邮件。我会尽快给您回复。谢谢，保持联系！

**Comment by woisdong** [comment]
您好！已收到您的邮件。

**Comment by miyazawa-kenya** [comment]
检测到视频源未能正常加载, 即将重置源.
获取云端源站失败. SyntaxError: Unexpected token < in JSON at position 0

软件重置一点就闪退，再启动还是一样报错

設備：macOS 15.0 Beta (24A5320a) MacBook Pro M2 Pro 2023

**Comment by woisdong** [comment]
您好！已收到您的邮件。

**Comment by IKUN-CJY** [comment]
大佬能不能增加一个弹幕与评论或者引入一下弹幕啊

**Comment by qishuai** [comment]
你好收到您的邮件。我会尽快给您回复。谢谢，保持联系！

**Comment by woisdong** [comment]
您好！已收到您的邮件。

**Comment by polo2023** [comment]
孩子看动画片非常好用，就是广告有些不健康。
大神能增加一个ai去广告的功能吗？谢谢！

**Comment by qishuai** [comment]
你好收到您的邮件。我会尽快给您回复。谢谢，保持联系！

**Comment by woisdong** [comment]
您好！已收到您的邮件。
